src/nifgen/system_tests/test_system_nifgen.py

```diff
@@ -50,3 +50,30 @@ def test_get_self_cal_last_date_and_time():
 
 def test_self_cal(session):
         session.self_cal()
+
+
+def test_standard_waveform(session):
+    session.output_mode = nifgen.OutputMode.NIFGEN_VAL_OUTPUT_FUNC
+    session.configure_standard_waveform(nifgen.Waveform.SINE, 2.0, 0.0, 0.0, 2000000)
+    session.output_enabled = True
+    session.trigger_mode = nifgen.TriggerMode.NIFGEN_VAL_CONTINUOUS
+    session.trigger_source = nifgen.TriggerSource.NIFGEN_VAL_IMMEDIATE
+    with session.initiate():
+        assert session.func_amplitude == 2.0
+        assert session.func_waveform == nifgen.Waveform.SINE
+        assert session.is_done() is False
+
+
+def test_frequency_list(session):
+    session.output_mode = nifgen.OutputMode.NIFGEN_VAL_OUTPUT_FREQ_LIST
+    session.clear_freq_list(-1)
+    duration_array = [0.01, 0.01, 0.01, 0.01, 0.01, 0.01, 0.01, 0.01, 0.01, 0.01]
+    frequency_array = [1000, 100900, 200800, 300700, 400600, 500500, 600400, 700300, 800200, 900100]
+    waveform_handle = session.create_freq_list(nifgen.Waveform.SQUARE, frequency_array, duration_array)
+    session.configure_freq_list(waveform_handle, 2.0, 0, 0)
+    session.trigger_mode = nifgen.TriggerMode.NIFGEN_VAL_CONTINUOUS
+    session.output_enabled = True
+    with session.initiate():
+        assert session.func_waveform == nifgen.Waveform.SQUARE
+    with session.initiate():
```

Comment:
What are you trying to test by initiating twice?
Why are you even initiating?

Remember: The system tests should only test that the Python calls make it to the right C function. You should not be testing driver behavior. See https://github.com/ni/nimi-python/wiki/Testing-Strategy